connect-inject/cleanup_resource_ent_test.go

```diff
@@ -89,6 +89,7 @@ func TestReconcile_ConsulNamespaces(t *testing.T) {
 			server.WaitForLeader(t)
```

Comment:
let's replace all the `WaitForLeader` with the `WaitForSerfCheck`? Unless you tried that and it didn't work.

Reply:
Yup good idea, I'll make the change!